sklearn/model_selection/_validation.py

```diff
@@ -726,14 +726,7 @@ def _fit_and_score(
     fit_params = _check_fit_params(X, fit_params, train)
 
     if parameters is not None:
-        # clone after setting parameters in case any parameters
-        # are estimators (like pipeline steps)
-        # because pipeline doesn't clone steps in fit
-        cloned_parameters = {}
-        for k, v in parameters.items():
-            cloned_parameters[k] = clone(v, safe=False)
-
-        estimator = estimator.set_params(**cloned_parameters)
+        estimator = estimator.set_params(**clone(parameters, safe=False))
```

Comment:
For my education: does this diff change behaviour? I can't tell. I think it doesn't because parameters were already being cloned and that cloning has now been moved into `clone`. So there should be no net change, but not sure.